upup/models/cloudup/resources/addons/eks-pod-identity-webhook.addons.k8s.io/v0.3.0.yaml.template

```diff
@@ -0,0 +1,173 @@
+# sourced from https://github.com/aws/amazon-eks-pod-identity-webhook/tree/b19c295a269ad9de50b10e3a9cdc9ec1f7d48a19/deploy
```

Comment:
this file should be named similar to the other addons

Reply:
Is it a name like `k8s-1.23.yaml.template`, for example?

Reply:
Yeah. Where the version is the minimum version supporting the manifests